### PR TITLE
[Tech] Added content header

### PR DESCRIPTION
**Pull request by bhlynsky**
Added heading to nested content using page title

### CODE REVIEW

File: docs/website/views/_layouts/popular-articles.njk [hunk 1]
```diff
@@ -2,8 +2,7 @@
 layout: content
 ---
 
-<h2 class="popular-articles-title">Welcome on Exadel authoring Kit for AEM</h2>
-<h3 class="popular-articles-subheader">Popular articles</h3>
+<h2 class="popular-articles-subheader">Welcome on Exadel authoring Kit for AEM</h2>
```
Comment (bhlynsky) [inline docs/website/views/_layouts/popular-articles.njk:5]: #### Considering previous change, this page wasn't looking well. So I thought I should include this update here. Let me know what you think about this changes

_Before update_


_After_

Comment (abarmina) [reply docs/website/views/_layouts/popular-articles.njk:5]: maybe we can remove this line `Welcome on Exadel authoring Kit for AEM` altogether?

File: docs/website/views/_layouts/content.njk [hunk 2]
```diff
@@ -1,11 +1,16 @@
...
 {% block content %}
   {% include 'navigation/header.njk' %}
   {% include 'navigation/sidebar.njk' %}
   <main class="esl-scrollable-content">
     <div class="content {{ containerCls or 'container markdown-container' }} {{ 'container-aside' if aside }}">
+      {% set fullPage = collections.content | findByPage(page) %}
+      {% set mainPageTitle = fullPage.mainParentPage.data.title %}
+      
+      {% if fullPage and fullPage.mainParentPage %}
+        <h1>{{ mainPageTitle }}</h1>
```
Comment (abarmina) [inline docs/website/views/_layouts/content.njk:12]: maybe we can not use the `mainPageTitle` variable here? as a suggestion
```suggestion
        <h1>{{ fullPage.mainParentPage.data.title }}</h1>
```

File: docs/website/src/page/articles-layout.less [hunk 3]
```diff
@@ -2,6 +2,7 @@
 
 .popular-articles {
   &-container {
+    padding: 0 !important; // override global markdown style for ul
```
Comment (abarmina) [inline docs/website/src/page/articles-layout.less:5]: can we override it without `!important`? with more specificity

**Review by abarmina — APPROVED** [review]

File: docs/website/views/_layouts/popular-articles.njk (hunk 1 above)
Comment (NastaLeo) [reply docs/website/views/_layouts/popular-articles.njk:5]: I agree with your suggestion)

**Review by NastaLeo — APPROVED** [review]

**Review by ala-n — APPROVED** [review]

**Pull request merged**